File: unity-client/Assets/2 - Scripts/entities/GiftCard.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class GiftCard 
{
    private string serial;
    private string code;

    public GiftCard(string serial, string code)
    {
        this.serial = serial;
        this.code = code;
    }

    public string Serial { get => serial; set => serial = value; }
    public string Code { get => code; set => code = value; }
}

File: ezy-roulette-http-server/src/main/java/com/example/ezyroulettehttpserver/entity/GiftCard.java
package com.example.ezyroulettehttpserver.entity;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document("gift_card")
public class GiftCard {
    @Id
    private Long id;

    private String serial;
    private String code;
    private int amount;
    private boolean activated;
}
